spec/core_functions/color/blackness.hrx

```diff
@@ -14,6 +14,17 @@ a {
   b: 100%;
 }
 
+<===> max/warning
+DEPRECATION WARNING: Individual color-channel functions defined globally or in the color module are deprecated in favor of the new color.channel() function.
+
+Recommendation: color.channel(black, blackness)
```

Comment:
I'm still not sure how this would work for a named color like `black`. 

`channel($color, $channel, $space: null)`

"If channel is not the name of a channel in color, throw an error." The channels here would be red, green, and blue, right? Does some conversion need to happen first?

Reply:
@dvdherron I am not sure. Since the channel is `blackness`, I thought this would be hwb() but maybe it is assumed to be RGB unless the $space is defined as that third param.